terraform/digitalocean/main.tf

```diff
@@ -82,6 +83,30 @@ module "admin_cert" {
   ca_private_key_pem = "${module.ca.ca_private_key_pem}"
 }
 
+module "docker_daemon_certs" {
+  source                = "github.com/Capgemini/tf_tls//docker/daemon"
```

Comment:
could these be done in 1-shot by just pointing to /docker and putting a TF file in the top level directory that calls down to both daemon+client to avoid repeating yourself here?